docs/readmes/howtos/inbound_roaming.md

```diff
@@ -0,0 +1,212 @@
+---
+id: inbound_roaming
+title: Inbound Roaming
+hide_title: true
+---
+
+*Last Updated: 4/08/2021*
+
+# Inbound Roaming
+
+Inbound Roaming allows subscribers belonging to other operators (roaming
+subscribers) to use Magma served network.
+
+Inbound Roaming requires Magma operator to reach agreements with
+other operators to have direct connectivity to their HSS through `S6a`
+(diameter) and PGW through `S8` (gtp) interfaces. VPN is suggested to reach
+those roaming services but is out of scope of Magma.
+
+## Architecture
+
+Currently, we support two Architectures:
+- **Non Federated + Roaming**: where local subscribers are stored in
+  SubscriberDb and roaming subscribers use a remote Federated Gateway to reach HSS/PGW
+- **Federated + Roaming**: where local Subscribers use local Federated Gateway to reach
+  HSS/PCRF/OCS and roam Subscribers use a remote Federated Gateway to reach HSS/PGW.
+
+The example below shows a possible architecture for a Non Federated +
+roaming case. As you can see in the picture below, VPN is suggested to
+reach to the user plane at the remote PGW.
+
+![Magma events table](assets/lte/inbound_roaming_architecture_non_federated.png?raw=true "Non Federated Inbound Roaming")
+
+Other configurations like SubscriberDb with PCRF/OCS to handle local
+subscribers may also work but are not tested yet.
+
+Configurations with SubscriberDb, local HSS and remote HSS all at the same
+time is not supported yet.
+
+## Pre Requisites
+
+Before starting to configure roaming setups, first you need to bring up a
+setup to handle your own/local subscribers.
+
+Even one of the architectures mentions `Non Federated`, Inbound Roaming
+requires a `Federated` setup to work. So the to configure Inbound Roaming
+you need:
+- Functioning [Or8cr](https://docs.magmacore.org/docs/orc8r/architecture_overview),
+- Install [Federatetion Gateway](https://docs.magmacore.org/docs/feg/deploy_intro) and,
+- Install [Access Gateway](https://docs.magmacoreorg/docs/lte/setup_deb).
+- Configure it as a [Federated Deployment](https://docs.magmacore.org/docs/feg/federated_FWA_setup_guide)
+- Make sure your setup is able to serve calls with your local subscribers
+  (in case of using subscriber DB you will need to set `"hss_relay_enabled":
+  false` on Federated LTE Network temporally to test this. Set it back to
+  `True` once tested!!)
+
+At the end of this you will have:
+- A Federated LTE Network with an LTE Gateway in it (AGW)
+- A Federation Network with a Federation Gateway in it (FEG)
+
+We will refer to them as `Local` Networks and Gateways to differentiate them
```

Comment:
`Local` and `roaming` casing should be consistent

Reply:
all lowered cased, except for `Inbound Roaming`